client/src/components/BlockEditor/isSelectionExpanded.js

```diff
@@ -0,0 +1,21 @@
+export function isSelectionExpanded(selection) {
```

Comment:
Why can't we use Slate's [`isExpanded`](https://docs.slatejs.org/slate-core/range#isexpanded)? Is selection not a range?

Reply:
Oh yeah we could use that. Not sure how I missed that. I'll switch it.

Reply:
https://github.com/oughtinc/mosaic/pull/145/commits/42d31e63e082390a02e4978d04a662b06905112e